include/aikido/rviz/InteractiveMarkerViewer.hpp

```diff
@@ -32,6 +34,15 @@ class InteractiveMarkerViewer {
   SkeletonMarkerPtr CreateSkeletonMarker(
     dart::dynamics::SkeletonPtr const &skeleton);
 
+  /// Visualizes a TSR.
+  /// \param tsr TSR constraint
+  /// \param nSamples Max number of samples to be used in visualization
+  /// \param basename Basename for markers
+  /// \return TSRMarkerPtr contains sampled frames of TSR.
+  TSRMarkerPtr addTSRMarker(
+    aikido::constraint::TSR const &tsr,
```

Comment:
I believe [the convention](https://github.com/personalrobotics/aikido/blob/master/STYLE.md#c-style) is to place `const` before type and space after `&` as `const aikido::constraint::TSR& tsr`.